guide/miscellaneous/parsing-mention-arguments.md

```diff
@@ -204,6 +208,75 @@ This will allow you to add proper checks for all your args, so that you can tell
 
 But this does not mark the end of the page. If you feel adventurous you can read on and learn how to use Regular Expressions to easily convert a mention into a user object in just two lines.
 
+### The ban command from the intro
```

Comment:
```suggestion
### Ban command
```
Since it's directly under `## Implementation`, this seems simpler. The intro is also mentioned in the following sentence.